src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import { Comment } from '../../types/Comment';
+import './CommentList.scss';
+
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+
+  <ul className="CommentList">
+    {comments.map(comment => (
+      <li key={comment.id}>
```

Comment:
move li inside CommentInfo component, and use key directly in the component calling, f.e
`<CommentInfo key={comment.id} comment={comment} />`